packages/atomic/src/components/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -52,6 +56,7 @@ export class AtomicSearchInterface {
   private unsubscribeSearchStatus: Unsubscribe = () => {};
   private hangingComponentsInitialization: InitializeEvent[] = [];
   private initialized = false;
+  private ignoreUrlManagerStateChanges = false;
```

Comment:
While fixing facet dependencies, we don't want the url manager's listener to be called.